dropwizard-benchmarks/src/main/java/io/dropwizard/benchmarks/jersey/ConstraintViolationBenchmark.java

```diff
@@ -0,0 +1,83 @@
+package io.dropwizard.benchmarks.jersey;
+
+import com.google.common.collect.Iterators;
+import io.dropwizard.jersey.validation.ConstraintMessage;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Valid;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.executable.ExecutableValidator;
+import javax.ws.rs.HeaderParam;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.commons.lang3.reflect.MethodUtils.getAccessibleMethod;
+
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@State(Scope.Benchmark)
+public class ConstraintViolationBenchmark {
+
+    private ConstraintViolation<ConstraintViolationBenchmark> paramViolation;
+    private ConstraintViolation<ConstraintViolationBenchmark> objViolation;
+
+    public String paramFunc(@HeaderParam("cheese") @NotEmpty String secretSauce) {
```

Comment:
Maybe we could we create a static class `Resource` and move both these methods there?
It will separate the benchmark from the test data.